### PR TITLE
[Runtime Enhence] Extend long input tokens length

### PR DESCRIPTION
**Pull request by Zhenzhong1**
## Type of Change

New feature
Bug fixed
Doc refined

## Description

* limit the maximum tokens lengths for all models.
* set the scratch_size_ratio

## Expected Behavior & Potential Risk

N/A

## How has this PR been tested?

https://inteltf-jenk.sh.intel.com/job/neural_speed_extension/84/
https://inteltf-jenk.sh.intel.com/job/neural_speed_extension/85/

## Dependency Change?

N/A

### CODE REVIEW

File: neural_speed/models/model_utils/model_utils.cpp [hunk 1]
```diff
@@ -911,7 +911,7 @@ struct model_context* model_init_from_file(const char* path_model, struct model_
   }
   ctx->cont_batching = params.cont_batching;
   ctx->generation_conf = params.gen_conf;
-  ctx->model_scratch_enlarge_scale = params.model_scratch_enlarge_scale;
+  ctx->model_scratch_size_ratio = params.model_scratch_size_ratio;
```
Comment (zhentaoyu) [inline neural_speed/models/model_utils/model_utils.cpp:914]: just a advice
```suggestion
ctx->model_scratch_size_ratio = ctx->batch_size > 1 ? params.model_scratch_size_ratio * params.max_request_num * params.beam_size : params.model_scratch_size_ratio
```

Comment (zhentaoyu) [reply neural_speed/models/model_utils/model_utils.cpp:914]: cc yishen @DDEle for some suggestions.

Comment (Zhenzhong1) [reply neural_speed/models/model_utils/model_utils.cpp]: modified.

**Comment by Zhenzhong1** [comment]
https://inteltf-jenk.sh.intel.com/job/neural_speed_extension/76/

**Review by a32543254 — APPROVED** [review]
LGTM

**Comment by Zhenzhong1** [comment]
https://inteltf-jenk.sh.intel.com/job/neural_speed_extension/82/

**Review by intellinjun — APPROVED** [review]

**Pull request merged**